js/impress.js

```diff
@@ -2323,6 +2323,8 @@
 
             if ( consoleWindow && !consoleWindow.closed ) {
                 consoleWindow.focus();
+                triggerEvent(root, 'impress:console:open', {});
```

Comment:
Please remember to run `node build.js` before submitting a PR. As far as I can tell, this `js/impress.js` now contains stuff that isn't at all part of this PR. This is more confusing than simply forgetting to re-build at all.